src/algorithms/controllers/msort_arr_bup.js

```diff
@@ -62,16 +58,8 @@ function isMergeExpanded() {
 }
 
 // -------------------------------------------------------------------------------
-
 // Define helper functions
-// without javascript Closure arguements (IE 'global variables')
-// -----------------------------------------------------------------------------------
-
-function assert(condition, message) {
-  if (!condition) {
-    throw new Error(message || 'Assertion failed');
-  }
-}
+// -------------------------------------------------------------------------------
 
 const highlight = (vis, index, isPrimaryColor = true) => {
```

Comment:
Possibly change isPrimaryColor to color so that different colours can be added in future

Reply:
Problem resolved